### PR TITLE
Specify Patch Target Using File

### PR DESCRIPTION
**Pull request by seanmalloy**
Use the YAML file to specify the patch target instead using the CLI
parameters. This avoids string parsing required for patching objects in
the v1 API group.

### CODE REVIEW

**Comment by seanmalloy** [comment]
@vinny-sabatini and @sanbornick if you like this approach I will submit a follow up PR to also do this for the delete action, and also do the final code clean up in the follow up PR.

Let me know what you think. Thanks!

**Review by sanbornick — COMMENTED** [review]
yes, my wish _has_ come true 😍

What's up with passing in the resource body in both `--patch` and the `data=` in the call to `run_oc`? Should the patch parameter and the type parameter come after `-f - ` or should those come before?

**Review by vinny-sabatini — APPROVED** [review]
LGTM, I'll wait for @sanbornick to review before merging

**Pull request merged**
